.github/workflows/smoke-test.yaml

```diff
@@ -0,0 +1,21 @@
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+name: smoke-test
```

Comment:
After this is rebased onto master can we add this job to the CI workflow (I think my last statement here was misleading).

Reply:
yes, I agree let's combine with `ci`.